lib/core/macros.nim

```diff
@@ -708,9 +708,12 @@ proc `$`*(node: NimNode): string {.compileTime.} =
 proc ident*(name: string): NimNode {.compileTime,inline.} = newIdentNode(name)
   ## Create a new ident node from a string
 
-iterator children*(n: NimNode): NimNode {.inline.}=
-  for i in 0 .. high(n):
-    yield n[i]
+iterator items*(n: NimNode): NimNode {.inline.} =
+  if n[0] != nil:
```

Comment:
@transfuturist nnkType does not belong here, from ast.nim, this node is an atom. Are you thinking of nnkTypeDef or nnkTypeSection?
```
    nkType,               # node is used for its typ field
```